CONTRIBUTING.md

```diff
@@ -210,6 +210,124 @@ the "actual" output, then re-run `cargo test` and make sure everything passes.
 
 Congrats on the new lint!
 
+#### Adding a witness
+
+**Witnesses** are a new, unstable feature of `cargo-semver-checks` that let us create witness
+code, a minimal compilable example of potential downstream breakage\. They are configured via the
```

Comment:
Let's explain the new term directly instead of making a circular definition first.
```suggestion
**Witnesses** are a new, unstable feature of `cargo-semver-checks` that let us create
minimal compile-able examples of potential downstream breakage\. They are configured via the
```